src/test/java/seedu/teachstack/logic/parser/SetWeakThresholdCommandParserTest.java

```diff
@@ -13,17 +14,24 @@ public class SetWeakThresholdCommandParserTest {
 
     @Test
     public void parse_setweakCommand() throws ParseException {
-        String args = "B";
+        String args = " g/B";
```

Comment:
Do you think you should use PREFIX_GRADE instead?

Reply:
To be deferred (group agreement)